make/_shared/oci-image/01_mod.mk

```diff
@@ -170,11 +170,11 @@ $(oci_push_targets): oci-push-%: oci-build-% | $(NEEDS_CRANE) $(NEEDS_COSIGN) $(
 ifneq ($(RELEASE_DRYRUN),true)
 	if $(CRANE) image digest $(oci_$*_image_name)@$(call oci_digest,$*) >/dev/null 2>&1; then \
 		echo "Digest already exists, will retag without resigning."; \
-		$(call oci_push_command,$*); \
+		$(call oci_push_command,$*) \
```

Comment:
I tried releasing v0.13.1 and the GitHub Action for releasing [failed](https://github.com/cert-manager/approver-policy/actions/runs/8437051805/job/23105931379) with the following:

```
bash: -c: line 3: syntax error near unexpected token `;;'
bash: -c: line 3: `	/home/runner/work/approver-policy/approver-policy/_bin/tools/crane push "_bin/scratch/image/oci-layout-manager.v0.13.1" "quay.io/jetstack/cert-manager-approver-policy:v0.13.1";; \'
make[1]: *** [make/_shared/oci-image/01_mod.mk:171: oci-push-manager] Error 2
make[1]: Leaving directory '/home/runner/work/approver-policy/approver-policy'
make: *** [make/02_mod.mk:36: release] Error 2
```

I guess I should have merged this "self-upgrade" before releasing...